Solution

denyschemeriakov opened the pull request.

alex-glotov67 reviewed (approved):
Very nice, but you can do this with one chain of methods and without any extra variables.
like
return `sourseString.split(';').map()x=> x.split(';')... ` end go on!
even without cycle for!

denyschemeriakov commented:
> Very nice, but you can do this with one chain of methods and without any extra variables.
> like
> return `sourseString.split(';').map()x=> x.split(';')... ` end go on!
> even without cycle for!

Cool solution! Thanks!)
